js/client-chat.js

```diff
@@ -747,8 +747,13 @@
 				var targets = target.split(',');
 				var formatTargeting = false;
 				var formats = {};
+				var gens = {};
 				for (var i = 1, len = targets.length; i < len; i++) {
-					formats[toId(targets[i])] = 1;
+					if (targets[i].length === 4 && targets[i].indexOf('gen') === 0) {
```

Comment:
I'll be honest, I missed that when I was looking through javascript string functions.